redfish-core/lib/assembly.hpp

```diff
@@ -365,9 +438,97 @@ inline void setAssemblylocationIndicators(
         {
             setLocationIndicatorActive(asyncResp, assembly, iter->second);
         }
+  
+        auto iter2 =
+            oemIndicatorMap.find(std::to_string(assemblyIndex));
+
+        if (iter2 != oemIndicatorMap.end())
+        {
+            std::optional<nlohmann::json> openbmc;
+            if (!json_util::readJson(iter2->second, asyncResp->res, "OpenBMC", openbmc))
+            {
+                BMCWEB_LOG_ERROR << "Property Value Format Error ";
+                messages::propertyValueFormatError(
+                                asyncResp->res, *openbmc,
+                                "OpenBMC");
+                return;
+            }
+
+            if (!openbmc)
+            {
+                BMCWEB_LOG_ERROR << "Property Missing ";
+                messages::propertyMissing(asyncResp->res, "OpenBMC");
+                return;
+            }
+
+            std::optional<bool> readytoremove;
+            if (!json_util::readJson(*openbmc, asyncResp->res,
+                                     "ReadyToRemove", readytoremove))
+            {
+                return;
```

Comment:
Can you add a 

                messages::propertyValueFormatError(
                                asyncResp->res, *readytoremove,
                                "ReadyToRemove");
                                
                                Here?

Reply:
@tomippolito this comment then I think I am good

Reply:
@tomippolito this comment then I think I am good